hazelcast/src/main/java/com/hazelcast/internal/nearcache/impl/store/AbstractNearCacheRecordStore.java

```diff
@@ -334,23 +334,29 @@ public boolean remove(K key) {
             record = removeRecord(key);
             if (record != null && record.getRecordState() == READ_PERMITTED) {
                 removed = true;
+                nearCacheStats.incrementInvalidations();
```

Comment:
Should't we call this `incrementEntryRemovalCount`? Other reasons can also be the cause to call this `remove` method like like expirations.